src/main/scala/com/fulcrumgenomics/umi/UmiConsensusCaller.scala

```diff
@@ -159,7 +158,7 @@ trait UmiConsensusCaller[C <: SimpleRead] {
 
   // vars to track how many reads meet various fates
   private var _totalReads: Long = 0
-  private val filteredReads = new SimpleCounter[String]()
+  private[umi] val _filteredReads = new SimpleCounter[String]()
```

Comment:
Why now `private[umi]`?  It looks like all accesses are still within `UmiConsensusCaller`.